store/src/metrics.rs

```diff
@@ -159,7 +158,7 @@ impl<S> ceramic_api::AccessInterestStore for StoreMetricsMiddleware<S>
 where
     S: ceramic_api::AccessInterestStore,
 {
-    async fn insert(&self, key: Interest) -> Result<bool> {
+    async fn insert(&self, key: Interest) -> anyhow::Result<bool> {
```

Comment:
Since we're in store, why does this not return a store error?

Reply:
This trait is from the API crate and we're just implementing it here. Could define some enum for that crate (or a generic ceramic one) but I didn't think it was necessary here since we just convert all errors to 500 right now.